### DISCUSSION TITLE
Brief fade out (and in) when stopping/starting

### DISCUSSION THREAD

**Discussion by poldie** [MusicPlayerDaemon/MPD#1497] [Ideas]
There's often a pop when stopping playback.   If the last few (literally 5 or so) samples were faded out it would prevent this.  Same for starting playback, fading in.

**Comment by britracer** [comment]
I agree this would be a really great enhancement.

**Comment by jarnos** [comment]
I get pops when I start playback and shortly after stop playback from headless server where the MPD daemon is running.  That is when I am using the ALSA output of the server for playback. (I could not setup Pulseaudio and maybe I don't need it.) I have a separate mpd user that runs the mpd daemon and also also pulseaudio seems to be running as mpd user for some reason.

I think maybe fading does not remove the pop sound. Maybe the sound is due to some hardware power switching of the audio output.
